test/PushNotificationTests/APITests.cpp

```diff
@@ -58,6 +58,7 @@ namespace Test::PushNotifications
             try
             {
                 TP::AddPackage_WindowsAppSDKFramework(); // Installs WASfwk
+                TP::AddPackage_DynamicDependencyLifetimeManager(); // Required when running the test app unpackaged.
```

Comment:
Gonna need the main package too (that has the DataStore)
If you install WASfwk you should always install the trifecta - WASfwk, WASmain, WASddlm

Reply:
Done